### PR TITLE
Adapt to separating of TrailingCommaInLiteral rule

### PR DESCRIPTION
**Pull request by ypresto**
See:
https://github.com/bbatsov/rubocop/pull/5307
https://github.com/bbatsov/rubocop/pull/5405

### CODE REVIEW

File: config/rubocop.yml [hunk 1]
```diff
@@ -36,5 +36,8 @@ Style/StringLiteralsInInterpolation:
 Style/TrailingCommaInArguments:
   Enabled: false
 
-Style/TrailingCommaInLiteral:
+Style/TrailingCommaInArrayLiteral:
+  Enabled: false
+
+Style/TrailingCommaInHashLiteral:
```
Comment (elim) [inline config/rubocop.yml:42]: @ypresto Thanks for your contribution! But it just seems a bit oddly.

The `Style/TrailingCommaInArrayLiteral` and `Style/TrailingCommaInHashLiteral` cops are introduced on RuboCop 0.53.0, but we use the RuboCop v0.52.1 via onkcop.

* https://github.com/bbatsov/rubocop/blob/master/CHANGELOG.md#0530-2018-03-05
* https://github.com/codetakt/rubocop-codetakt/blob/v0.8.3/rubocop-codetakt.gemspec#L15
* https://github.com/onk/onkcop/blob/v0.52.1.1/onkcop.gemspec#L22

What version of RuboCop are you using?

**Comment by ypresto** [comment]
oh sorry, I'm running wrong rubocop executable (was 0.54.0).
Keeping this open until onkcop updates its rubocop dependency.

**Comment by elim** [comment]
Acknowledged 👍

**Comment by elim** [comment]
Thank you for your efforts.
This repository has been completely rebuilt. In fact, it is a fork of onkcop. As such, it now includes a corresponding change. 1942502794aa03c410a27b60fe6c52a2f5caf59c

**Pull request closed without merging**